README.md

```diff
@@ -206,6 +206,7 @@ Please update these lists when proposing NIPs introducing new event kinds.
 | `l`               | label, label namespace               | annotations          | [32](32.md)                           |
 | `L`               | label namespace                      | --                   | [32](32.md)                           |
 | `m`               | MIME type                            | --                   | [94](94.md)                           |
+| `q`               | event id (hex)                       | relay URL, marker    | [18](18.md)                           |
```

Comment:
`q` tags don't need any marker since they're a separate tag.

```suggestion
| `q`               | event id (hex)                       | relay URL    | [18](18.md)                           |
```

Reply:
Oops. Yes, thanks!